.github/workflows/CI-notebook-text.yml

```diff
@@ -0,0 +1,111 @@
+name: CI Text Notebooks
```

Comment:
Would it be desirable to add another yaml file `ci-e2e-notebooks-text` similar to vision?

Reply:
yes, this will be done in a future PR.  Trying to keep these PRs smaller.